src/data/agile-quiz.ts

```diff
@@ -290,6 +290,36 @@ const agileQuiz = [
     Explanation:
       "The Kanban method does not prescribe roles or meetings unlike other frameworks such as Scrum.",
     Link: "https://www.freecodecamp.org/news/being-agile-kanban-vs-scrum/"
+  },
+  {
+    Question: "In a Scrum team, who needs to comply with the Definition of Done?",
+    Answer: "The Scrum Team",
+    Distractor1: "The Product Owner",
+    Distractor2: "The Scrum Master",
+    Distractor3: "The Product Owner and the Scrum Master",
+    Explanation:
+      "The Developers are required to conform to the Definition of Done. If there are multiple Scrum Teams working together on a product, they must mutually define and comply with the same Definition of Done.",
+    Link: "https://scrumguides.org/scrum-guide.html"
+  },
+  {
+    Question: "Who is accountable for establishing Scrum and Scrum Team's effectiveness?",
+    Answer: "The Scrum Master",
+    Distractor1: "The Scrum Team",
+    Distractor2: "The Product Owner and the Scrum Master",
+    Distractor3: "The Product Owner",
+    Explanation:
+      "Scrum Master is responsible for establishing scrum within team & at organization level. They achieve this by enabling the Scrum Team to improve its practices, within scrum framework and help them understand Scrum theory & practices.",
```

Comment:
I think it is fine to leave off the last sentence. If users want to learn more, then they can always click on the learn more link in the quiz. 
```suggestion
      " The Scrum Master is responsible for establishing scrum within the team & organizational level.",
```